core/src/date-input/date-input.css

```diff
@@ -59,6 +59,16 @@
 	width: 32px;
 	outline: none;
 }
+.light .DayPicker-Day--disabled {
+	background-color: var(--gray-1);
+	opacity: 0.5	;
```

Comment:
why there is a tab at the end of the line

Reply:
also we don't prefer opacity, let me change it to a fixed color (var gray something)